pkg/cmd/services.go

```diff
@@ -150,10 +151,31 @@ func findServicePlanByNameAndClass(scClient versioned.Interface, planName, servi
 		}
 	}
 
-	return nil, nil
+	return nil, errors.New("failed to find serviceplan with associated with the serviceclass " + serviceClassName)
+}
```

Comment:
one extra `with` snuck in.
failed to find serviceplan associated with the serviceclass